phpseclib/Math/BigInteger.php

```diff
@@ -919,7 +919,7 @@ function _add($x_value, $x_negative, $y_value, $y_negative)
             $carry = $sum >= self::$maxDigit2; // eg. floor($sum / 2**52); only possible values (in any base) are 0 and 1
             $sum = $carry ? $sum - self::$maxDigit2 : $sum;
 
-            $temp = self::$base === 26 ? intval($sum / 0x4000000) : ($sum >> 31);
+            $temp = self::$base === 26 ? (int) ($sum / 0x4000000) : ($sum >> 31);
```

Comment:
This should be fine, but should be in 1.0 too. How do we prevent people from using intval? Is there a sniff?